src/constants/defaultStatuses.js

```diff
@@ -0,0 +1,11 @@
+const PENDING = 1;
+const APPROVED = 2;
+const REJECTED = 3;
+
+const DEFAULT_STATUS = {
```

Comment:
Please use 'creatEnum' util from the @deip/toolbox package for enums.
Do not use such generic names as `DEFAULT_STATUS`. There are can be a bunch of states between `PENDING` and `APPROVED`, i.e. `CERTIFYING`, `IN REVIEW` and other. Use specific names for enums i.e. `CONTRACT_AGREEMENT_STATUS`